.github/workflows/ci.yaml

```diff
@@ -212,6 +212,9 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Free up runner disk space
+        uses: ./.github/actions/free-disk-space
```

Comment:
i've seen this action take ~10 minutes some times, have you seen the build fail due to low disk space?

Reply:
removed